src/System.Private.ServiceModel/tests/Scenarios/SelfHostWcfService/ConfigBackup.xml

```diff
@@ -0,0 +1,176 @@
+﻿<configuration>
+  <!-- 
+    This is a copy of the old app.config for reference.
+    My intention is to replace all the XML config with code.
```

Comment:
From an earlier conversations it was called out that having config was easier. If the others feel OK then I think it is fine. But being able to pull from config is definitely helpful in some cases, However given k doesn't have config support it might be ok to have the server also avoid config.